core/src/main/java/discord4j/core/spec/MessageCreateSpecGenerator.java

```diff
@@ -62,7 +63,14 @@ default List<MessageCreateFields.FileSpoiler> fileSpoilers() {
 
     Possible<AllowedMentions> allowedMentions();
 
-    Possible<Snowflake> messageReference();
+    /**
+     * @deprecated this is just a helper for the reply format in this spec you need {@link MessageReferenceData}
+     */
+    @Deprecated
+    Possible<Snowflake> messageReferenceId();
+
+    @Experimental
```

Comment:
This shouldn't be experimental if the other method is deprecated

Reply:
Removed